scripts/travis/upload_coverage.sh

```diff
@@ -12,4 +12,4 @@ if [[ "${UPSTREAM_VERSION}" != "${LOCAL_VERSION}" ]]; then
   echo "WARN: version ${UPSTREAM_VERSION} of the codecov upload script is available."
 fi
 
-/usr/bin/env bash scripts/travis/codecov
+/usr/bin/env bash scripts/travis/codecov "$@"
```

Comment:
I thought this file was being replaced with a github action, is that still in progress @algojack

Reply:
different codecov was added in https://github.com/algorand/go-algorand/pull/3997/files but the old one wasn't removed yet. Are we ok to remove the old one or is there something missing still?

Reply:
oh this PR just modifies the .codecov.yml file to add a "nightly" tag to the coverage reports generated nightly, because nightly tests will get fuller coverage than regular PRs.

Reply:
i guess @cce pointed out that while it was added, the upload action was not added in circleci. We still need that.

Reply:
oh I found it — https://github.com/algorand/go-algorand/issues/3962 OK I will take a look at the indexer change and put it in here

Reply:
Switching to the orb from the bash uploader didn't work, because the codecov uploader does not support Linux with ARM. So I think we have to stick with the codecov script we have checked in until they fix that.
![image](https://user-images.githubusercontent.com/51567/175437526-d663654d-61be-424a-b05a-d5dec9807287.png)
https://app.circleci.com/pipelines/github/algorand/go-algorand/7388/workflows/e8b6d40c-cae7-4a72-9547-8e2ae8cd5c01/jobs/131023